hibernate-rx-core/src/main/java/org/hibernate/rx/RxHibernateSession.java

```diff
@@ -22,4 +24,6 @@
 
 	RxActionQueue getRxActionQueue();
 
+    CompletionStage<Void> rxFlush();
```

Comment:
Is this method necessary? Couldn't we just use the one in RxSession? No need to change, i'm just wondering

Reply:
Maybe not. I was confused as to why we had two session classes.